frontera/contrib/backends/remote/codecs/json.py

```diff
@@ -3,29 +3,59 @@
 """
 from __future__ import absolute_import
 import json
+import six
 from base64 import b64decode, b64encode
 from frontera.core.codec import BaseDecoder, BaseEncoder
-from w3lib.util import to_unicode, to_native_str
-from frontera.utils.misc import dict_to_unicode, dict_to_bytes
+from w3lib.util import to_unicode, to_bytes
+
+
+def _encode_recursively(obj):
+    """
+    recursively encodes an object to unicode and store its type
```

Comment:
@kmike Thanks for the review. 
I had previously named this method `convert`: https://github.com/scrapinghub/frontera/pull/233/commits/33347d9a9f0c5a9e732612350b320ca09309baa8#diff-6cd48fdd254557ffc1b50d60a0323364R12. 

Side Note: I think I should start contributing to scrapy. There's a lot I can learn from your reviews :)